CI's `Tests / Docs / Doctest (pull_request)` throws an exception

# Bug report

### Bug description:

I'm not sure if this is caused by my PR. But it seems to be caused by the CI service running
```
Run xvfb-run make -C Doc/ PYTHON=../python SPHINXERRORHANDLING="-W --keep-going" doctest
  xvfb-run make -C Doc/ PYTHON=../python SPHINXERRORHANDLING="-W --keep-going" doctest
  shell: /usr/bin/bash -e {0}
  env:
    FORCE_COLOR: 1
make: Entering directory '/home/runner/work/cpython/cpython/Doc'
make[1]: Entering directory '/home/runner/work/cpython/cpython/Doc'
mkdir -p build

Missing the required blurb or sphinx-build tools.
Please run 'make venv' to install local copies.

make[1]: *** [Makefile:55: build] Error 1
make[1]: Leaving directory '/home/runner/work/cpython/cpython/Doc'
Testing of doctests in the sources finished, look at the results in build/doctest/output.txt
make: *** [Makefile:134: doctest] Error 1
make: Leaving directory '/home/runner/work/cpython/cpython/Doc'
```
from:
https://github.com/python/cpython/actions/runs/11423310997/job/31782368635?pr=125719
https://github.com/python/cpython/actions/runs/11424088507/job/31784009966?pr=125750
https://github.com/python/cpython/actions/runs/11423945175/job/31783706551?pr=125748
https://github.com/python/cpython/actions/runs/11424088507/job/31784009966?pr=125750
https://github.com/python/cpython/actions/runs/11425133485/job/31786988131?pr=125752

### CPython versions tested on:

CPython main branch

### Operating systems tested on:

_No response_

--- a/Makefile.pre.in
+++ b/Makefile.pre.in
@@ -176,7 +176,7 @@ EXPORTSYMS=	@EXPORTSYMS@
 EXPORTSFROM=	@EXPORTSFROM@
 
 # Executable suffix (.exe on Windows and Mac OS X)
-EXE=		@EXE_SUFFIX@
+EXE=		@EXEEXT@
 BUILDEXE=	@BUILDEXEEXT@
 
 # Name of the patch file to apply for app store compliance

--- a/configure
+++ b/configure
@@ -7296,11 +7296,11 @@ then :
   withval=$with_suffix;
 	case $with_suffix in #(
   no) :
-    EXE_SUFFIX= ;; #(
+    EXEEXT= ;; #(
   yes) :
-    EXE_SUFFIX=.exe ;; #(
+    EXEEXT=.exe ;; #(
   *) :
-    EXE_SUFFIX=$with_suffix
+    EXEEXT=$with_suffix
    ;;
 esac
 
@@ -7308,20 +7308,20 @@ else $as_nop
 
   case $ac_sys_system/$ac_sys_emscripten_target in #(
   Emscripten/browser*) :
-    EXE_SUFFIX=.js ;; #(
+    EXEEXT=.js ;; #(
   Emscripten/node*) :
-    EXE_SUFFIX=.js ;; #(
+    EXEEXT=.js ;; #(
   WASI/*) :
-    EXE_SUFFIX=.wasm ;; #(
+    EXEEXT=.wasm ;; #(
   *) :
-    EXE_SUFFIX=
+    EXEEXT=
    ;;
 esac
 
 fi
 
-{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $EXE_SUFFIX" >&5
-printf "%s\n" "$EXE_SUFFIX" >&6; }
+{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $EXEEXT" >&5
+printf "%s\n" "$EXEEXT" >&6; }
 
 # Test whether we're running on a non-case-sensitive system, in which
 # case we give a warning if no ext is given
@@ -7332,7 +7332,7 @@ if test ! -d CaseSensitiveTestDir; then
 mkdir CaseSensitiveTestDir
 fi
 
-if test -d casesensitivetestdir && test -z "$EXE_SUFFIX"
+if test -d casesensitivetestdir && test -z "$EXEEXT"
 then
     { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 printf "%s\n" "yes" >&6; }
@@ -7340,7 +7340,7 @@ printf "%s\n" "yes" >&6; }
 else
 	{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: no" >&5
 printf "%s\n" "no" >&6; }
-	BUILDEXEEXT=$EXE_SUFFIX
+	BUILDEXEEXT=$EXEEXT
 fi
 rmdir CaseSensitiveTestDir
 

--- a/configure.ac
+++ b/configure.ac
@@ -1325,19 +1325,19 @@ AC_ARG_WITH([suffix],
             [AS_HELP_STRING([--with-suffix=SUFFIX], [set executable suffix to SUFFIX (default is empty, yes is mapped to '.exe')])],
 [
 	AS_CASE([$with_suffix],
-    [no], [EXE_SUFFIX=],
-    [yes], [EXE_SUFFIX=.exe],
-    [EXE_SUFFIX=$with_suffix]
+    [no], [EXEEXT=],
+    [yes], [EXEEXT=.exe],
+    [EXEEXT=$with_suffix]
   )
 ], [
   AS_CASE([$ac_sys_system/$ac_sys_emscripten_target],
-    [Emscripten/browser*], [EXE_SUFFIX=.js],
-    [Emscripten/node*], [EXE_SUFFIX=.js],
-    [WASI/*], [EXE_SUFFIX=.wasm],
-    [EXE_SUFFIX=]
+    [Emscripten/browser*], [EXEEXT=.js],
+    [Emscripten/node*], [EXEEXT=.js],
+    [WASI/*], [EXEEXT=.wasm],
+    [EXEEXT=]
   )
 ])
-AC_MSG_RESULT([$EXE_SUFFIX])
+AC_MSG_RESULT([$EXEEXT])
 
 # Test whether we're running on a non-case-sensitive system, in which
 # case we give a warning if no ext is given
@@ -1347,13 +1347,13 @@ if test ! -d CaseSensitiveTestDir; then
 mkdir CaseSensitiveTestDir
 fi
 
-if test -d casesensitivetestdir && test -z "$EXE_SUFFIX"
+if test -d casesensitivetestdir && test -z "$EXEEXT"
 then
     AC_MSG_RESULT([yes])
     BUILDEXEEXT=.exe
 else
 	AC_MSG_RESULT([no])
-	BUILDEXEEXT=$EXE_SUFFIX
+	BUILDEXEEXT=$EXEEXT
 fi
 rmdir CaseSensitiveTestDir
 

